_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -108,7 +108,11 @@ FIXME:none as of Sunday; harding to update or delete this section Tuesday
 
 - [BOLTs #807][] Fail channel in case of high-S remote signature reception FIXME:dongcarl
 
-- [Eclair #1545][] Add blockchain watchdog FIXME:Xeyko
+- [Eclair #1545][] adds a blockchain watchdog to detect eclipse attacks. Eclair
+  fetches block headers from multiple third-party sources to countercheck the
+  best chain provided by its peers. As described in [Newsletter #101][], an
+  attacker could use an eclipse attack to hide a malicious channel closing
+  attempt until the dispute period expires.
```

Comment:
First PR description I did. :)

I've been considering to switch the order of sentence two and three, i.e. first explain the attack, then the defense. Any preferences on that?

Reply:
Seems good to me as-is :)